client/components/foldable-card/index.jsx

```diff
@@ -85,10 +86,12 @@ var FoldableCard = React.createClass( {
 			);
 		}
 		if ( this.props.children ) {
-			let iconSize = 24;
+			const iconSize = 24;
+			const screenReaderDefaultText = ( typeof this.translate === 'function' ) ? this.translate( 'More' ) : '';
```

Comment:
Would it be possible to abstract this somehow? Maybe this text should always be passed in the props? My concern is that this approach makes things kind of messy.